### PR TITLE
added #80 websocket vuex component

### PR DESCRIPTION
**Pull request by rohitgupta274509**
please review and approve it.

### CODE REVIEW

File: src/components/Settings/ProfileSetting.vue [hunk 1]
```diff
@@ -71,6 +71,7 @@
   .p-float-label {
     margin-bottom: 10px;
   }
+
```
Comment (wildone) [inline src/components/Settings/ProfileSetting.vue:74]: probs no need for this line :D

**Review by wildone — APPROVED** [review]

**Review by Shivam-apna — APPROVED** [review]

File: src/components/Settings/ProfileSetting.vue (hunk 1 above)
Comment (wildone) [reply src/components/Settings/ProfileSetting.vue:74]: ah i see cool then yours win.

Comment (wildone) [reply src/components/Settings/ProfileSetting.vue:74]: but this file does not exist anymore, you need to merge develop into this branch.

Comment (rohitgupta274509) [reply src/components/Settings/ProfileSetting.vue:74]: conflict resolved

**Review by Rahul139-stack — APPROVED** [review]

**Review by ankitvats1 — APPROVED** [review]

**Pull request merged**